xml/ha_rear.xml

```diff
@@ -541,11 +522,56 @@
     adjust it according to your setup:
    </para>
 <screen>BACKUP=NSR
-    OUTPUT=ISO
-    BACKUP_URL=nfs://<replaceable>host.&exampledomain;/path/to/rear/backup</replaceable>
-    OUTPUT_URL=nfs://<replaceable>host.&exampledomain;/path/to/rear/backup</replaceable>
-    NSRSERVER=<replaceable>backupserver.&exampledomain;</replaceable>
-    RETENTION_TIME="Month"</screen>
+OUTPUT=ISO
+BACKUP_URL=nfs://<replaceable>host.&exampledomain;/path/to/rear/backup</replaceable>
+OUTPUT_URL=nfs://<replaceable>host.&exampledomain;/path/to/rear/backup</replaceable>
+NSRSERVER=<replaceable>backupserver.&exampledomain;</replaceable>
+RETENTION_TIME="Month"</screen>
+   <para>
+    For more information about supported third-party backup tools, see
+    <command>man rear</command>, section <citetitle>BACKUP SOFTWARE INTEGRATION</citetitle>.
+   </para>
+  </example>
+
+  <example xml:id="ex-ha-rear-config-multipath">
+    <title>Backing up multipath devices</title>
+    <para>
+     By default, &rear; ignores file systems on multipath devices because it assumes they are
+     on shared storage, not part of the local system. You can make &rear; include these
```

Comment:
I am not at all a multipath / SAN storage expert
but I think usually multipath / SAN storage volumes
are not shared.
Even regardless whether or not they are shared,
the meaning of 'shared' is not what is meant here.
What is meant is that those storage volumes are
"not part of the local system" because they are
on remote storage - i.e. 'remote' versus 'local'
is what is meant here - and by default ReaR reasonably
assumes that only the local system got destroyed and
only that needs to be recreated.
It would be wrong to overwrite by default intact data
on remote storage with a needless backup restore.